mappings/net/minecraft/client/gui/screen/container/AbstractContainerScreen.mapping

```diff
@@ -0,0 +1,16 @@
+CLASS id net/minecraft/client/gui/screen/container/AbstractContainerScreen
```

Comment:
is this replacing `ContainerScreen` ? I would prefer the more concise `ContainerScreen` mapping if so (one of the conventions for these mappings is being concise)

Reply:
Wait

Reply:
`ContainerScreen` was not a thing so this is not replacing

Reply:
but if you want i can change

Reply:
It was already mapped as this I believe
Still, `ContainerScreen` is more concise

Reply:
there fixed